test/functional/rpc_psbt.py

```diff
@@ -394,6 +394,14 @@ def run_test(self):
         # We don't care about the decode result, but decoding must succeed.
         self.nodes[0].decodepsbt(double_processed_psbt["psbt"])
 
+        # Make sure unsafe inputs are included if specified
+        self.nodes[2].createwallet(wallet_name="unsafe")
+        wunsafe = self.nodes[2].get_wallet_rpc("unsafe")
+        self.nodes[0].sendtoaddress(wunsafe.getnewaddress(), 2)
+        self.sync_mempools()
+        assert_raises_rpc_error(-4, "Insufficient funds", wunsafe.walletcreatefundedpsbt, [], [{self.nodes[0].getnewaddress():1}])
+        wunsafe.walletcreatefundedpsbt([], [{self.nodes[0].getnewaddress():1}], 0, {"include_unsafe": True})
```

Comment:
style nit if you need to retouch (same on previous line)
```suggestion
        wunsafe.walletcreatefundedpsbt([], [{self.nodes[0].getnewaddress(): 1}], 0, {"include_unsafe": True})
```

Reply:
The rest of that file doesn't put a whitespace in that case, this is why I used this format to stay coherent across the file, but if you think it makes sense to fix I should change it everywhere in the file, right?

Reply:
I wouldn't worry about it apart from the new code and even then it's a pico nit.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/21359/commits/2bf0b1e7ee28dbc06aebf832c0480d257dad7294